lib/Runtime/Library/JavascriptString.cpp

```diff
@@ -2134,37 +2134,51 @@ namespace Js
     Var JavascriptString::ToCaseCore(JavascriptString* pThis, ToCase toCase)
     {
         charcount_t count = pThis->GetLength();
+        charcount_t countToCase = count;
 
+        const wchar_t* inStr = pThis->GetString();
+        const wchar_t* inStrLim = inStr + count;
+        const wchar_t* i = inStr;
+
+        // Try to find out the chars that do not need casing
+        if (toCase == ToUpper)
+        {
+            while (i < inStrLim) {
+                if (*i >= 224 || (*i <= 118 && *i >= 97)) break;
+                i++;
+            }
+        } else {
+            Assert(toCase == ToLower);
+            while (i < inStrLim) {
+                if (*i >= 257 || (*i <= 204 && *i >= 192) || (*i <= 90 && *i >= 65)) break;
```

Comment:
Initially, yes, but I found out the char for 256 is okay too. So, I thought why not include it too. 
If you prefer, I could change this to 256, I have no preference for one way or the other.